### DISCUSSION TITLE
Compile Error : No default Base file specified

### DISCUSSION THREAD

**Discussion by hyobokkim** [mark-wiemer/ahkpp#426] [Q&A] [answered]
ahk2exe.exe was downloaded through install-ahk2exe.ahk in the UX folder.

![image](https://github.com/mark-wiemer-org/ahkpp/assets/69405195/5879182d-4acd-4daa-bc13-ceb35a27b762)

When compiling with `ctrl`+`shift`+`F9` in vscode, the following error occurs.

![image](https://github.com/mark-wiemer-org/ahkpp/assets/69405195/b1d04cca-d6e6-4f97-9673-59e4d289568c)

2024-04-07T00:15:36.515Z Command failed: "C:/Program Files/AutoHotkey/Compiler/Ahk2Exe.exe"  /in "f:\vscode_projects\hello.ahk" /out "f:\vscode_projects\hello.exe"   
Failed to compile: f:\vscode_projects\hello.ahk

How do I specify a Base File ('v2.0.12 U32 AutoHotkey32.exe' OR 'v2.0.12 U64 AutoHotkey64.exe' ) on the command line? Is there a way to control it with parameters in settings.json?

I have limited knowledge so I would like some help.

**Comment by mark-wiemer** [comment, marked as answer]
You can set it in VS Code settings (Ctrl + comma) under the setting "compile base file"
![image](https://github.com/mark-wiemer-org/ahkpp/assets/7833360/7ad6a177-3b5b-4fb2-86aa-500ffee8b141)

You can also use the GUI via Ahk2Exe, accessible from the AutoHotkey Dash:
![image](https://github.com/mark-wiemer-org/ahkpp/assets/7833360/5f4d20e2-e17e-492f-ac67-67135c0dfd04)

![image](https://github.com/mark-wiemer-org/ahkpp/assets/7833360/b4f73bb0-8468-408b-bb06-1f40e06311ee)

**Reply by hyobokkim** [reply]
Thank you so much! Solved.
